### PR TITLE
Ticket 4876

### PR DESCRIPTION
**Pull request by rerpha**
### Description of work

*Add your own description here*

### To test

*Which ticket does this PR fix?*

### Acceptance criteria

*List the acceptance criteria for the PR*

---

#### Code Review

- [ ] Is the code of an acceptable quality?
- [ ] Has the author taken into account the multi-threaded nature of the code?
- [ ] Have the changes been documented in the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev). If so, do they describe the changes appropriately?
- [ ] Has the [manual system tests spreadsheet](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Manual-system-tests) been updated?

### Functional Tests

- [ ] Do changes function as described? Add comments below that describe the tests performed.

### Final steps

- [ ] Reviewer has updated the submodule in the main EPICS repo? See **Reviewing work for the subModules of EPICS** in the [Git workflow](https://github.com/ISISComputingGroup/ibex_developers_manual/wiki/Git-workflow) page for details.
- [ ] Reviewer has moved the [release notes](https://github.com/ISISComputingGroup/IBEX/wiki/ReleaseNotes_Dev) entry for this ticket in the "Changes merged into master" section

### CODE REVIEW

File: scripts/instrument_status.py [hunk 1]
```diff
@@ -25,10 +25,10 @@
...
     def error(self, message):
-        print ("ERROR: {0}".format(message))
+        print("ERROR: {0}".format(message))
 
     def info(self, message):
-        print ("INFO : {0}".format(message))
+        print("INFO : {0}".format(message))
 
 
 LOG = Log()
```
[the remark this answers is not in the document]
Comment (JamesKingWork) [reply scripts/instrument_status.py:34]: Or could make it an f-string

File: scripts/set_instrument_list.py [hunk 2]
```diff
@@ -92,7 +92,7 @@ def set_instlist(instruments_list, pv_address):
     if result != new_value:
```
Comment (JamesKingWork) [inline scripts/set_instrument_list.py:92]: Same problem with compress_and_hex and dehex_and_decompress functions in this file. This is duplicate code so should just be imported from the other file, or the other way around.

**Review by JamesKingWork — DISMISSED** [review]
Some subtle issues with how bytes and strings are handled, but otherwise good.

The log file generator script (line 72) has backslashes in the string, either make these double backslashes to escape or make the description into a raw string

File: scripts/set_instrument_list.py (hunk 2 above)
Comment (rerpha) [reply scripts/set_instrument_list.py:92]: i'll just import the one from the server_common dir, that method is more documented

**Comment by rerpha** [comment]
I have imported compress_and_hex and dehex_and_decompress from server_common, the only thing that's different is the type check but I don't think this'll break anything.

**Comment by JamesKingWork** [comment]
> I have imported compress_and_hex and dehex_and_decompress from server_common, the only thing that's different is the type check but I don't think this'll break anything.

Even better, less duplication!

**Review by JamesKingWork — APPROVED** [review]
Looks good

**Pull request merged**